pkg/kv/kvserver/metrics.go

```diff
@@ -96,6 +97,25 @@ var (
 		Unit:        metric.Unit_COUNT,
 	}
 
+	metaRaftFlowsReplicate = metric.Metadata{
```

Comment:
Could also do the "template" [trick](https://github.com/cockroachdb/cockroach/blob/0dfde1529854d4b707e1f6d7894943f8c690eb45/pkg/kv/kvserver/kvflowcontrol/kvflowcontroller/kvflowcontroller_metrics.go#L155) from `kvflowcontrol`.

Reply:
Seems fine to keep it like this, the other metric declarations are pretty verbose.